website/pages/how-to-guides/how-to-load-infrastructure-data-into-athena.md

```diff
@@ -0,0 +1,303 @@
+---
+title: How to Load Infrastructure Data into Athena
+tag: tutorial
+description: Load CloudQuery source data into AWS Athena.  
+author: hermanschaaf
+---
+
+import { HowToGuideHeader } from "../../components/HowToGuideHeader"
+
+<HowToGuideHeader/>
+
+# How to Load Infrastructure Data into Athena  
+
+## Introduction
+
+Athena is a serverless query service by AWS that allows you to query data in S3 using standard SQL. In this tutorial, we will show you how to load your cloud infrastructure data into S3 using CloudQuery and query it using Athena. This allows you to get fine-grained insights into your infrastructure data, all from the convenience of a serverless query environment running on AWS. 
+
+By the end of this post, you will be able to query your AWS infrastructure data using Athena:
+
+![Athena query editor](/images/how-to-guides/how-to-load-infrastructure-data-into-athena/athena-query-editor.png)
+
+Let's get started!
+
+## Steps
+
+### Step 1: Install CloudQuery
+
+To sync infrastructure data to S3, you will first need an installation of the CloudQuery CLI (or Docker image). See our [Quickstart](/docs/quickstart) for detailed instructions.
+
+### Step 2: Create a Bucket for the Data
+
+We will need to upload the sync results to S3 so that Athena can query them. We'll use the AWS CLI to do this in this tutorial, but you can also use the AWS web console or Terraform/CloudFormation if you prefer.
+
+First, we'll create a bucket to store the data:
+
+```bash copy
+export BUCKET_NAME=cloudquery-athena-example
+aws s3 mb s3://$BUCKET_NAME
```

Comment:
why do we do it like this? why export the variable (could do `BUCKET_NAME=value aws s3 mb ...`), or why have the variable at all? (`aws s3 mb s3://cloudquery-athena-example`).

Also since bucket names are globally unique this is guaranteed to fail, so I would include some kind of placeholder for the user to fill in.

Reply:
We do it because this variable is also used in a number of places lower down in the tutorial, and if you do this you can copy-paste the rest. But we can add a comment here that you need to change it to your desired bucket name?

Reply:
It's only on two more places, not like we're embedding it in the policy json or something. But sure, a comment in the bash telling the user to come up with their own bucket name would be neat :)

Reply:
Done 👍